### PR TITLE
Getting started with PHP Sessions

### LINKED ISSUE #2673: Getting Started with PHP Sessions
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of article
Getting Started with PHP Sessions

### Introduction paragraph (2-3 paragraphs):
PHP Sessions provide server side storage of user information by use of session variables and makes this information accessed from several pages throughout a website.
Sessions uniquely identify users via a Session Identifier which provides connection of that particular user to their stored data on the server.
This article intends to cover how to create a Session, how to update a Session and how to destroy a Session.

### Key takeaways: 
1.What is a Sessions and why sessions are used.
2. How to create a Session.
3. How to access a Session.
4. Updating a PHP Session.
5. How to destroy a Session.

### References: 
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Neema-2016**
### Article checklist
This fixes #2673 

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
@Neema-2016 
Good afternoon and thank you for submitting your article for publishing. It has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. 🚀

**Comment by Neema-2016** [comment]
Hey @WanjaMIKE ,
Forgive my ignorance kindly. But I note the "Still in progress? convert to draft" at the top right hand side. Is it meant for me? Got me a little concerned pending my article review. Haha please enlighten me.

 TIA.

**Comment by Neema-2016** [comment]
*noticed

**Comment by WanjaMIKE** [comment]
@Neema-2016 if there is no change you need to make, just ignore it. Thanks

**Comment by Neema-2016** [comment]
well noted, Thank you.

**Comment by WanjaMIKE** [comment]
@Neema-2016 You copy pasted the whole article in the same file - Now you have two articles in the index.md. Avoid such errors in futures

File: content/authors/neema-muganga/index.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Neema Muganga is a Tech enthusiast who enjoys creating web applications using ReactJS and Django tools. Networking with peers, exploring and learning new tech skills particularly interests her.
```
Comment (WanjaMIKE) [inline content/authors/neema-muganga/index.md:1]: Please check how other contributors have created their profile - index.md. You need to use the same style. You can find this information in the `content/authors/` folder.

**Comment by Neema-2016** [comment]
> @Neema-2016 You copy pasted the whole article in the same file - Now you have two articles in the index.md. Avoid such errors in futures

Hey @WanjaMIKE an honest mistake, my bad. I just fixed it

**Comment by ninjaginja** [comment]
plagiarism check completed - all good
[2120420-6879535487f45bb70c48-1626796242932.pdf](https://github.com/section-engineering-education/engineering-education/files/6849667/2120420-6879535487f45bb70c48-1626796242932.pdf)

**Comment by ahmadmardeni1** [comment]
Welcome to Section EngEd🎉 @Neema-2016 
After some careful consideration, it struck us that this topic may be a bit over-saturated throughout other blog sites and official documentation.

We typically refrain from publishing content that is covered widely on the net or other blogs. As we're more interested in original, practitioner-focused content that takes a deeper dive into programming-centric concepts.

We believe this is the best way for students to build a great portfolio (for potential employers) is by building what does not exist and what can provide the most value.

You are more than welcome to continue working on this one by flushing it out a bit more, you can add any idea you feel that can make your article unique and different.

**Comment by Neema-2016** [comment]
> Welcome to Section EngEd🎉 @Neema-2016
> After some careful consideration, it struck us that this topic may be a bit over-saturated throughout other blog sites and official documentation.
> 
> We typically refrain from publishing content that is covered widely on the net or other blogs. As we're more interested in original, practitioner-focused content that takes a deeper dive into programming-centric concepts.
> 
> We believe this is the best way for students to build a great portfolio (for potential employers) is by building what does not exist and what can provide the most value.
> 
> You are more than welcome to continue working on this one by flushing it out a bit more, you can add any idea you feel that can make your article unique and different.

@ahmadmardeni1 Thank you and it is well noted. Let me see how I could work around perfecting on this one.

**Comment by Neema-2016** [comment]
Hey @WanjaMIKE I just made the enhancements. Kindly check it out. Thank you.

**Comment by Neema-2016** [comment]
Hey, @WanjaMIKE just updated the article. Kindly check. Thank you.

**Comment by ahmadmardeni1** [comment]
Welcome to Section EngEd and congrats on your first article.🎉 @Neema-2016 
 
Great review, thank you @WanjaMIKE 🚀

**Pull request merged**